src/EventTeachingService.cpp

```diff
@@ -344,10 +344,13 @@ void EventTeachingService::handleReadEventVariable(const VlcbMessage *msg, unsig
   {
     // Return number of EVs. This may be dynamic in other implementations.
     controller->sendMessageWithNN(OPC_NEVAL, eventIndex, evnum, module_config->EE_NUM_EVS);
-    for (byte i = 1; i <= module_config->EE_NUM_EVS; i++)
+    if (!controller->getModuleConfig()->fcuCompatible)
```

Comment:
You can replace "controller->getModuleConfig()" with "module_config".

Reply:
Of course!  I'm not sure why I did that.  I'll change it